app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -22,6 +22,148 @@
 }
 
 .new-guide {
+<<<<<<< HEAD
+
+  .guide-creation-nav {
+    $guide-nav-color: #5E9546;
+    background: $guide-nav-color;
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    /* the ul containing the links */
+    ul {
+      list-style: none;
+      margin-left: 0;
+      padding-top: 5px;
+      padding-left: 0;
+      padding-right: 0;
+      text-align: center;
+      white-space: nowrap;
+    }
+
+    li {
+      position: relative;
+      display: inline-block;
+    }
+
+    li a {
+      color: white;
+      font-size: 1.25em;
+      font-weight: 600;
+      vertical-align: middle;
+      color: rgba(255,255,255,0.7);
+      padding: 10px 10px 10px 35px;
+
+      &.disabled {
+        pointer-events: none;
+      }
+
+      &:hover {
+        color: rgba(255,255,255,0.9);
+      }
+
+      &::before {
+        position: absolute;
+        top: 10px;
+        left: 10px;
+        $size: 12px;
+        width: $size; height: $size;
+        content: "";
+        @include box-shadow(
+            0 0 0 2px $guide-nav-color,
+            0 0 0 4px #D9E9D2);
+        border-radius: 50%;
+      }
+
+      &.check::after {
+        @include manual-font-awesome('\f00c');
+        position: absolute;
+        top: 4px;
+        left: 8px;
+        font-size: 22px;
+        color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.